src/frontends/common/include/openvino/frontend/node_context.hpp

```diff
@@ -48,6 +48,11 @@ class FRONTEND_API NodeContext {
         FRONT_END_NOT_IMPLEMENTED(get_input);
     }
 
+    /// \brief Returns the input by reference. The reference value can be changed by consuming operation
+    virtual Output<Node> get_input_by_reference(int idx) const {
```

Comment:
It is not clear to me from the description what this function is used for? Can we have something similar in PT FE that we can use it?

Reply:
discussed offline. This is mostly about removing deps from TF FE in tokenizers.
some conversion extensions use this method which handle tensors by reference. Since we want to move tokenizers to common FE API we have to expand this API.
It may be useful for PT FE if you have tensor references.

Reply:
Please better describe this

Reply:
Let me make it in the next PR please. Hope not critical